FoundationsMasteryTests/CustomStringTests.cs

```diff
@@ -1,46 +1,119 @@
 ﻿using System;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FoundationsMastery;
+using System.Collections.Generic;
 
 namespace FoundationsMasteryTests
 {
+    
     [TestClass]
     public class CustomStringTests
     {
+
         [TestMethod]
         public void EnsureICanCreateAnInstance()
         {
-        }
+            CustomString customString = new CustomString();
 
+        Assert.IsNotNull(customString);
+        }
+        
         [TestMethod]
         public void EnsureICanPassInACharEnumerable()
         {
             char[] mychars = new char[] { 'a', 'b', 'c' };
             CustomString myString = new CustomString(mychars);
 
-            //int expected_length = ??
-            //int actual_length = ??
+            int expected_length = 3;
+            int actual_length = mychars.Length;
```

Comment:
We don't want `mychars.Length`, we want `myString.Length` - implement the getter for the Length property and this should pass